### PR TITLE
edit login form

### PR DESCRIPTION
**Pull request by Minhtu12394**

### CODE REVIEW

**Review by nguyenvu1892 — COMMENTED** [review]
oke

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Admin::UsersController < ApplicationController
-  before_action :ensure_admin
 
   def index
-
+    @admin = User.where(role: [:admin])
```
Comment (huongnguyenmta) [inline app/controllers/admin/users_controller.rb]: define `scope` in User model

File: app/models/user.rb [hunk 2]
```diff
@@ -7,4 +7,10 @@ class User < ApplicationRecord
   has_many :courses, through: :user_course
 
   enum role: [:admin, :trainer, :trainee]
+
+  before_save {self.email = email.downcase}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
+            format: {with: VALID_EMAIL_REGEX}
```
Comment (huongnguyenmta) [inline app/models/user.rb]: indent

File: app/models/user.rb [hunk 3]
```diff
@@ -7,4 +7,10 @@ class User < ApplicationRecord
   has_many :courses, through: :user_course
 
   enum role: [:admin, :trainer, :trainee]
+
+  before_save {self.email = email.downcase}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
```
Comment (huongnguyenmta) [inline app/models/user.rb]: using `gem config`

File: app/views/admin/users/index.html.erb [hunk 4]
```diff
@@ -1 +1,21 @@
+<h1>Admin</h1>
```
Comment (huongnguyenmta) [inline app/views/admin/users/index.html.erb]: I18n

**Pull request merged**